scripts/start.sh

```diff
@@ -15,19 +15,33 @@ ctx logger info "Starting SimpleHTTPServer"
 nohup python -m SimpleHTTPServer ${port} > /dev/null 2>&1 &
 echo $! > ${PID_FILE}
 
-ctx logger info "Waiting for server to launch"
+ctx logger info "Waiting for server to launch on port ${port}"
+url="http://localhost:${port}"
+
+server_is_up() {
+	if which wget >/dev/null; then
+		if wget $url >/dev/null; then
+			return 0
+		fi
+	else
```

Comment:
I would do elif which curl just for the sake of making sure that if they're both non-existing, it will fail before it continues.